### PR TITLE
docs: add v15 changes and update info

### PR DESCRIPTION
**Pull request by bob-watson**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Doesn't describe update to v15

Issue Number: N/A

## What is the new behavior?
Describes the update to v15

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/update-to-version-15.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+*  [angular/angular changelog](https://github.com/angular/angular/blob/main/CHANGELOG.md)
+*  [angular/angular-cli changelog](https://github.com/angular/angular-cli/blob/main/CHANGELOG.md)
+*  [angular/components changelog](https://github.com/angular/components/blob/main/CHANGELOG.md)
+
+Information about updating Angular applications to v14 is archived at [Update to version 14](/guide/update-to-version-14).
+
+<a id="new-features"></a>
+
+## New features in Angular v15
```
Comment (crisbeto) [inline aio/content/guide/update-to-version-15.md:23]: Should this mention the directive composition API/host directives?

Comment (bob-watson) [reply aio/content/guide/update-to-version-15.md:23]: Updated in latest commit.

File: aio/content/guide/update-to-version-15.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+<a id="v15-bc-09"></a>
+
+### `RouterOutlet` instantiates the component after change detection
+
+Before v15, during navigation, `RouterOutlet` instantiated the component being activated immediately. [PR #46554](https://github.com/angular/angular/pull/46554)
+
+In v15, the component is not instantiated until after change detection runs.
+This change could affect tests that do not trigger change detection after a router navigation.
+Less common, this could affect production code that relies on the exact timing of component availability.
```
[the remark this answers is not in the document]
Comment (bob-watson) [reply aio/content/guide/update-to-version-15.md:162]: Updated in latest commit.

**Review by mgechev — APPROVED** [review]

**Review by AndrewKushnir — APPROVED** [review]

**Review by jessicajaniuk — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 032a488d027587c42ecfec9098cec33757403d8a.

**Pull request closed without merging**